src/TextHelper.php

```diff
@@ -43,6 +43,10 @@ public static function trim_characters($text, $num_chars = 60, $more = '&hellip;
      */
     public static function trim_words($text, $num_words = 55, $more = null, $allowed_tags = 'p a span b i br blockquote')
     {
+        if (null === $text) {
```

Comment:
We could also return here when the text is empty, right?

Reply:
as in 

`if(empty($text)) {`

@gchtr ? That would short circuit the function a lot earlier in some cases. Which sounds like a good idea.

Reply:
Yeah, exactly like that.

My only question is: does this change the logic so that `$more` will not be displayed, when it was before, even with empty text? And is this an acceptable change? In my opinion, this would be okay.

In the future, this function will probably get proper parameter types as well (`trim_words(?string $text, ...)`, so a check for `empty()` would catch empty text and null values, while other values would be discarded earlier with an error.